examples/tutorials/3_space-embedding.py

```diff
@@ -23,11 +23,6 @@
 This file shows how to grow space-embedded neurons with DeNSE.
 """
 
-import numpy as np
```

Comment:
If you change files in ``examples/tutorials``, please check that they are still rendered properly in the documentation (my guess is that this is not the case anymore, so please either restore or update documentation ;)